src/main/kotlin/blackjack/controller/BlackJackGame.kt

```diff
@@ -0,0 +1,50 @@
+package blackjack.controller
+
+import blackjack.domain.Deck
+import blackjack.domain.GameTable
+import blackjack.domain.User
+import blackjack.view.InputView
+import blackjack.view.ResultView
+
+class BlackJackGame(
+    private val gameTable: GameTable,
+    private val inputView: InputView,
+    private val resultView: ResultView,
+) {
+    fun start() {
+        val names = inputView.inputNames()
+        val deck = Deck.create()
+        val users = names.map { User.create(name = it) }
+
+        val initCardReceivedUsers = gameTable.dealInitCard(users, deck)
+
+        println()
+        resultView.printInitCardReceive(initCardReceivedUsers)
+        initCardReceivedUsers.forEach { resultView.printUserCards(user = it, printScore = false) }
+
+        val allCardReceivedUsers =
+            initCardReceivedUsers.map { user ->
+                var currentUser = user
+                while (true) {
+                    if (!currentUser.canReceiveCard()) {
```

Comment:
```suggestion
                while (currentUser.canReceiveCard() && inputView.inputReceiveMoreCard()) {
```
위처럼 뎁스를 줄일수 있는 방법을 고민해보면 좋을거같네요!

Reply:
고민할거리 주셔서 감사해요 남재님 :)
반영해보도록 할게요🙂